Changes as of 12/18/2019

z-q-y opened the pull request:
- Implements dropout for several NN models, of which
  + NN_ensemble is functional
  + NN is still undergoing cosmetic changes
  + Other folders are experimental and will soon be deleted
- more updates to follow

z-q-y committed:
Changes as of 12/19/2019; implements dropout for several NN models

more updates to follow

ktran9891 commented:
We were not supposed to try any dropouts on anything except for the simple NN. No need to commit files and delete them afterwards; just don't commit them in the first place.

Splits on the dropout NN look better now, but as you probably know the test results aren't done yet. And there is no NLL calculation. Also, the [dropout branch of CGCNN](https://github.com/ulissigroup/cgcnn/tree/dropout) still does not look like it has been updated correctly.

z-q-y committed:
Removed superfluous files and implemented other advice offered by KTran

To Kevin: the CGCNN repos dropout branch is now updated; please let me know if you want the dropout version of NN_ensemble here as well - I left that out this time

z-q-y commented:
Please let me know if any of you need the dropout version of NN_ensemble here; I left that out this time though it is working. Happy Holidays!

z-q-y committed:
Some  changed files were previously untracked

ktran9891 commented:
Regarding CGCNN:  [this](https://github.com/ulissigroup/cgcnn/commit/d135532e620461b9c636937d4259dad598bc1ef8) type of change is what we were looking for. It is effectively the same as what you did, but without the changes to leaky ReLus and with relying more heavily on torch classes that they already made.

Regarding using dropout:  [this](https://github.com/ulissigroup/uncertainty_benchmarking/commit/b8be8da22da94132598577879d9bb106e777d2b6) is what we were looking for.
